compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/text/KeyMapping.desktop.kt

```diff
@@ -33,7 +33,7 @@ private val _platformDefaultKeyMapping: KeyMapping =
 internal fun createPlatformDefaultKeyMapping(platform: DesktopPlatform): KeyMapping {
     return when (platform) {
         DesktopPlatform.MacOS -> createMacosDefaultKeyMapping()
-        else -> defaultKeyMapping
+        else -> defaultSkikoKeyMapping
```

Comment:
Shouldn't we just modify `defaultKeyMapping` in common?

Reply:
@MatkovIvan I'd ve glad to I just thought that this is less recommended because we need to merge back and to coordinate any changes - no matter how small - with google team.

Reply:
If the code is platform specific, better to keep it in our source set only.

If the code isn't platform specific - better to change it in commonMain and upstream.

Key mapping usually is platform specific, even if we have some default. We can't tell if it is needed for Android. `defaultKeyMapping` was added only because we didn't want to investigate platform behaviours.